incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -530,6 +529,18 @@ namespace if this annotation already exists. The two are mutually exclusive.
 We are considering replacing this with the standard
 `app.kubernetes.io/managed-by` label in the future.
 
+#### hnc.x-k8s.io/excluded-namespace (label on namespaces)
+
+This label is supposed to be added to the excluded namespaces such as
+`kube-system`, `kube-public`, and `hnc-system` so that when HNC validating
+webhook services are down, operations in these excluded namespaces won't be
+affected.
+
```

Comment:
State what the defaults are (as of March 2021, in case it changes) and maybe link to the relevant file and line. Also mention that HNC manifests exclude `hnc-system` by default, but that you need to set up the rest yourself.